File: sample-legacy/src/main/java/com/example/legacy/StringUtils.java
package com.example.legacy;

public class StringUtils {
    private StringUtils() {
        // Utility class
    }

    public static boolean isEmpty(String str) {
        return str == null || str.trim().length() == 0;
    }

    public static String capitalize(String str) {
        if (isEmpty(str)) {
            return str;
        }
        return str.substring(0, 1).toUpperCase() + str.substring(1).toLowerCase();
    }
}

File: archived-runs/run-2-simpletodo-task-success/migrated-src/com/example/legacy/StringUtils.kt
package com.example.legacy

object StringUtils {
    fun isEmpty(str: String?): Boolean = str.isNullOrBlank()

    fun capitalize(str: String?): String {
        if (isEmpty(str)) {
            return str ?: ""
        }
        return str.substring(0, 1).toUpperCase() + str.substring(1).toLowerCase()
    }
}